src/products/iOS_products/review_iOSstrings.md

```diff
@@ -1,70 +1,70 @@
 
 ### Acceptable changes
 
-Changes to attributes like `tool-version` and `build-num` are expected, they happen every time you change the version of Xcode.
+Changes to attributes like `tool-version` and `build-num` are expected, they happen every time you change the version of Xcode. Seeing `datatype="plaintext"` move to the end of `file original` is also acceptable, such as [here](https://github.com/mozilla-l10n/firefoxios-l10n/pull/195/files#diff-6f29eb799be1b575316c0187d69a38fce3c63e27e5a22eb180b338ee177e9caeL3529).
 
 Note that these will not appear as new strings for localizers and they will not need to touch them.
 
 Changes to comments (`<note>`) are irrelevant in terms of string updates, so they’re also OK.
 
-### Potentially problematic changes (removals)
+### Potentially problematic changes
 
-Then you start scrolling down, checking if the new strings make sense: the first one you find is `NSCameraUsageDescription`.
+Then you start scrolling down, checking if the new strings make sense: the first one you find is `CreditCard.DisplayCard.ExpiresLabel.v115`.
 
-You mostly need to pay attention to the strings removed. Double-check each time that iOS team clearly realizes that they won’t be able to release updates to v5 (or any v5.x release) once strings are exported.
-
-For example: `Menu.NightModeAction.Title` is removed. If it’s used in v5, and the iOS team plans to release an update to v5 in the future, localizations will miss that string.
+Things to look out for:
+* Unclear strings and missing localization comments: the best way to identify them is to translate the strings, only having the string and comment as context (not the entire file, or the associated commit where the string landed). For example: is the word used both a noun and a verb in English? Is the ID clear enough to give context (e.g. `buttonLabel`)?
+* String changes without new IDs. IDs in XLIFF files are stored in the `trans-unit` attribute; for example, the ID of [this string](https://github.com/mozilla-l10n/firefoxios-l10n/pull/192/files#diff-6f29eb799be1b575316c0187d69a38fce3c63e27e5a22eb180b338ee177e9caeR3201) is `CreditCard.DisplayCard.ExpiresLabel.v115`.
+* Duplicated strings.
+* Other general [localization issues](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html).
 
 ### Quickly review other locales
 
-Once you’ve checked `templates`, you can pick at least one locale and see what changes, as you can see [for example here](https://github.com/mozilla-l10n/firefoxios-l10n/pull/19/commits/a1c041a012e6996bd8bdaec88f21f3c34fe383b0).
+Once you’ve checked `templates`, you can pick at least one locale and see what changes, as you can see [for example here](https://github.com/mozilla-l10n/firefoxios-l10n/pull/192/files#diff-220a1dc4ddc01bbae6a176ba4122aa644042181c78a638b052f45462b758ca6f).
```

Comment:
There is one bit missing here: why do you want to check a locale, and what are you looking for?

Reply:
the main reason I can think of is that by checking an actual locale you can easily:
Check that there aren't entire files being removed, that are deleting existing translations that we would want to keep

there may be other cases but can't seem to come up with any now.

Reply:
Updated with more info as per comment

Reply:
Update: added more info from #251